tests/ui_tests/test_cluster_list_page.py

```diff
@@ -394,19 +394,32 @@ def test_run_upgrade_on_cluster_list_page(self, sdk_client_fs, app_fs):
             ), f"Cluster state should be {params['state']}"
 
     @pytest.mark.parametrize(
-        ("upgrade_name", "config", "hc_acl", "disclaimer_text"),
+        ("upgrade_name", "config", "hc_acl", "is_new_host", "disclaimer_text"),
         [
-            ("simple_upgrade", None, False, False),
-            ("upgrade_with_hc_acl", None, True, False),
-            ("upgrade_with_config", {"somestring2": "test"}, False, False),
-            ("upgrade_with_config_and_hc_acl", {"somestring2": "test"}, True, False),
-            ("upgrade_with_hc_acl_and_disclaimer", None, True, DISCLAIMER_TEXT),
-            ("upgrade_with_config_and_disclaimer", {"somestring2": "test"}, False, DISCLAIMER_TEXT),
-            ("upgrade_with_config_and_hc_acl_and_disclaimer", {"somestring2": "test"}, True, DISCLAIMER_TEXT),
+            ("simple_upgrade", None, False, False, False),
+            ("upgrade_with_hc_acl", None, True, False, False),
+            ("upgrade_with_config", {"somestring2": "test"}, False, False, False),
+            ("upgrade_with_config_and_hc_acl", {"somestring2": "test"}, True, False, False),
+            ("upgrade_with_hc_acl_and_disclaimer", None, True, False, DISCLAIMER_TEXT),
+            ("upgrade_with_config_and_disclaimer", {"somestring2": "test"}, False, False, DISCLAIMER_TEXT),
+            ("upgrade_with_config_and_hc_acl_and_disclaimer", {"somestring2": "test"}, True, False, DISCLAIMER_TEXT),
```

Comment:
maybe make it like 1 arg?
dict with key-values that are really required to be passed to `run_upgrade_in_cluster_row` as `run_upgrade_in_cluster_row(**upgrade_kwargs)`, then it will be easier to read (no need to connection arg position and name in the string above)

Reply:
not shure, for me it looks like this version is more readable

Reply:
okay, tomatos tomahtos)